chapter_notation/index_vn.md

```diff
@@ -78,14 +91,26 @@ The notation used throughout this book is summarized below.
 * $\prod$: Series multiplication
 -->
 
-*dịch đoạn phía trên*
+* $f(\cdot)$: một hàm số
+* $\log(\cdot)$: logarit tự nhiên
+* $\exp(\cdot)$: hàm $e$ mũ
+* $\mathbf{1}_\mathcal{X}$: The indicator function <!-- Thảo luận -->
+* $\mathbf{(\cdot)}^\top$: chuyển vị của một vector hoặc một ma trận
+* $\mathbf{X}^{-1}$: nghịch đảo của ma trận $\mathbf{X}$
+* $\odot$: tích Hadamard (theo từng thành phần)
+* $\lvert \mathcal{X} \rvert$: Cardinality (số phần tử) của tập $\mathcal{X}$
```

Comment:
Mình thấy trong từ điển thì `Cardinality` còn một nghĩa toán học nữa là `bản số`? Có ai biết nó có đúng nghĩa trong trường hợp này không nhỉ?

Reply:
Tiếng Việt nhiều tài liệu viết là `card`
```suggestion
* $\lvert \mathcal{X} \rvert$: card (số phần tử) của tập $\mathcal{X}$
```